model/invite.go

```diff
@@ -0,0 +1,31 @@
+package model
+
+import (
+	"errors"
+	"time"
+)
+
+// Invite is a representation of the invite model.
+// Token field is required for proper working.
+type Invite struct {
+	ID        string    `json:"id" bson:"_id"`
+	AppID     string    `json:"appId" bson:"appId"`
+	Token     string    `json:"token" bson:"token"`
+	Valid     bool      `json:"valid" bson:"valid"`
```

Comment:
this one is conflicts with 'Validate', should be something like 'archived' if this is a flag that set explicitly

Reply:
Renamed to `Archived`